include/libssh2_sftp.h

```diff
@@ -303,9 +303,19 @@ LIBSSH2_API int libssh2_sftp_rename_ex(LIBSSH2_SFTP *sftp,
                            LIBSSH2_SFTP_RENAME_ATOMIC | \
                            LIBSSH2_SFTP_RENAME_NATIVE)
 
+LIBSSH2_API int libssh2_sftp_posix_rename_ex(LIBSSH2_SFTP *sftp,
+                                             const char *source_filename,
+                                             unsigned int srouce_filename_len,
+                                             const char *dest_filename,
+                                             unsigned int dest_filename_len);
```

Comment:
Speaking of a new API, what about going with `size_t` sizes?

Reply:
Sounds good to me.